.github/workflows/check-openapi-updates.yml

```diff
@@ -0,0 +1,14 @@
+name: Check OpenAPI updates
+
+on:
+  schedule:
+    - cron: "*/5 * * * *" # every 5 minutes (for testing)
+    # - cron: "0 16 * * *" # every day at 4pm UTC
```

Comment:
I can't see how to run this from the PR, so I figured we could merge it, see it run, and replace these lines once it's working.